sdcm/cluster.py

```diff
@@ -3440,10 +3440,13 @@ def wait_for_init(self):
     def install_scylla_monitoring_prereqs(self, node):
         if node.is_rhel_like():
             prereqs_script = dedent("""
+                yum install -y wget
+                yum install -y unzip
                 yum install -y epel-release
-                yum install -y python-pip unzip wget
-                pip install --upgrade pip
-                pip install pyyaml
+                yum install -y python36
+                yum install -y python36-pip
+                python36 -m pip install --upgrade pip
```

Comment:
never run python using specific version binary, if it is python 3 it should be `python3`

Reply:
changed to python3